development_setup_wizard.py

```diff
@@ -0,0 +1,265 @@
+    hooray('Successfully built and started docker containers')
+
+
+def set_env_values(args):
+    if os.environ.get('CAMPHORIC_SETUP_WIZARD_RECREATE_SECRET_KEY') == 'no':
+        return
+
+    filename = './env/local/django.env'
+    contents = ''
+    with open(filename) as env_file:
+        contents = env_file.read().strip()
+
+    current_key_pair = ''
+    for line in re.split(r'[\r\n]+', contents):
+        if line.startswith('SECRET_KEY='):
+            current_key_pair = line
+            break
+
+    current_key = ''
+    if current_key_pair:
+        current_key = current_key_pair.split('=')[1]
+    else:
+        current_key = 'your-django-secret-key-here'
+        contents += '\nSECRET_KEY=your-django-secret-key-here'
+
+    if args.preserve:
+        return
+
+    if (current_key != 'your-django-secret-key-here'
+            and 'CAMPHORIC_SETUP_WIZARD_RECREATE_SECRET_KEY' not in os.environ):
+        answer = input(
+            f'it looks like there is already a SECRET_KEY in your {filename} file. '
+            'Do you want replace it with a newly generated one? [y/n] ')
+        if answer.lower() != 'y':
+            return
+
+    secret_key = get_secret_key()
+    contents = re.sub(r'SECRET_KEY=.*', f'SECRET_KEY={secret_key}', contents)
+    with open(filename, 'w') as env_file:
+        env_file.write(contents)
+    hooray(f'Successfully wrote secret key to {filename}')
+
+    # Rebuild if we added secret key
+    docker_compose_up(args)
+
+
+def get_secret_key():
+    cmd = (
+        'docker compose exec -T django python -c '
+        '"from django.core.management.utils import get_random_secret_key; '
+        'print(get_random_secret_key())"')
+    result = None
+    secret_key = None
+    stderr = None
+    try:
+        result = subprocess.run(cmd, shell=True, capture_output=True, check=True)
+    except subprocess.CalledProcessError as e:
+        stderr = e.stderr
+    else:
+        stderr = result.stderr
+        secret_key = (
+            result.stdout.decode().strip()
+            .replace("'", "\\'")
+            .replace("$", "_"))  # django-environ uses $ for interpolation
+
+    if not secret_key:
+        die(
+            "Could not successfully get secret key",
+            f"tried '{cmd}'"
+            f'\n{stderr.decode()}')
+    else:
+        hooray('Successfully got secret key')
+
+    return secret_key
+
+
+def create_django_superuser(args):
+    print(bold('Creating Django superuser using values from env/local/shared.env'))
+    try:
+        subprocess.run(
+            [
+                'docker', 'compose', 'exec', 'django',
+                'python', 'manage.py', 'createsuperuser', '--noinput'
+            ],
+            check=True)
+    except subprocess.CalledProcessError:
+        answer = input('Continue anyway? [y/n] ')
+        if answer.lower() != 'y':
+            die('Failed to create Django superuser')
+    hooray('Successfully created Django superuser')
```

Comment:
I adjusted the initial message here to reflect that the values are being read from the env file. We might want to consider making this optional, so the dev can choose their own interactively.

Also, the "Continue anyway?" prompt is to handle the case where the user already exists from a previous run of the setup and manage.py fails with "That username is already taken."